docs/sync-status.md

```diff
@@ -0,0 +1,27 @@
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# About DataHub Sync Status
+
+<FeatureAvailability/>
+
+When looking at metadata in DataHub, it's useful to know if the information you're looking at is relevant.
+Specifically, if metadata is stale, or hasn't been updated in a while, then you should consider refreshing that metadata
+using [metadata ingestion](./../metadata-ingestion/README.md) or [deleting](TODO) it if it no longer exists.
+
+## Sync Status Setup, Prerequisites, and Permissions
+
+The sync status feature is enabled by default and does not require any special setup.
+
+## Using Sync Status
+
+The DataHub UI will display the sync status in the top right corner of the page.
+
+![sync status](https://raw.githubusercontent.com/datahub-project/static-assets/master/imgs/sync-status.png)
```

Comment:
Small nit-pick for consistency with other guides & improved readability - please use HTML to embed images so we can size down (70%) & center-align: 

```
<p align="center">
  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/master/imgs/sync-status.png"/>
</p>
```